src/components/molecules/NavbarSchedule/NavbarSchedule.scss

```diff
@@ -0,0 +1,53 @@
+@import "scss/constants.scss";
```

Comment:
Note that while there are a bunch of things in these styles that likely need fixing up, they will also likely change a lot in the `Schedule` UI refactors in https://github.com/sparkletown/sparkle/pull/1302